addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -4,7 +4,7 @@
         <div class="pos-receipt p-2">
             <t t-set="showTaxGroupLabels" t-value="doesAnyOrderlineHaveTaxLabel()"/>
             <ReceiptHeader data="props.data.headerData" />
-            <OrderWidget lines="props.data.orderlines" t-slot-scope="scope">
+            <OrderWidget lines="props.data.orderlines" t-slot-scope="scope" generalNote="props.data.generalNote or ''" screenName.translate="RecieptScreen">
```

Comment:
```suggestion
            <OrderWidget lines="props.data.orderlines" t-slot-scope="scope" generalNote="props.data.generalNote or ''" screenName="RecieptScreen">
```

I don't think you want to translate the string "ReceiptScreen". It's a technical string we used in the app.